src/test/java/com/tabnine/plugin/HintTests.kt

```diff
@@ -1,89 +0,0 @@
-package com.tabnine.plugin
```

Comment:
why are these tests removed?

Reply:
Because they were shit tests, and I don't know how to create good tests for this 🤷